Themis/Views/Assessment/Modeling Exercise/Diagram Renderers/Class Diagram/UMLClassDiagramRenderer.swift

```diff
@@ -10,14 +10,23 @@ import Common
 import SharedModels
 
 struct UMLClassDiagramRenderer: UMLDiagramRenderer {
-    var context: GraphicsContext
+    var context: UMLGraphicsContext
     let canvasBounds: CGRect
-    
-    private let fontSize: CGFloat = 14
+    var fontSize: CGFloat
 
+    init(context: GraphicsContext, canvasBounds: CGRect, fontSize: CGFloat = 14) {
+        self.context = UMLGraphicsContext(context)
+        self.canvasBounds = canvasBounds
+        self.fontSize = fontSize
+    }
```

Comment:
We don't really need the initializer here, right? We could simply cast the context in the UMLRendererVM. Or do I miss something?
 
```suggestion
    }
```

Reply:
Right, I'll remove it 👍